src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -0,0 +1,300 @@
+<cd-loading-panel *ngIf="loading && !error"
+                  i18n>Loading configuration...</cd-loading-panel>
+<cd-alert-panel type="error"
+                *ngIf="loading && error"
+                i18n>The configuration could not be loaded.</cd-alert-panel>
+
+<div class="cd-col-form"
+     *ngIf="!loading && !error">
+  <ng-container [ngSwitch]="step">
+    <!-- Configuration step -->
+    <div *ngSwitchCase="1">
+      <form name="form"
+            #formDir="ngForm"
+            [formGroup]="configForm"
+            novalidate>
+        <div class="card">
+          <div class="card-header"
+               i18n>Step {{ step }} of 2: Telemetry report configuration</div>
+          <div class="card-body">
+            <p i18n>The telemetry module sends anonymous data about the cluster back to the Ceph developers
+              to help understand how Ceph is used and what problems users may be experiencing.<br/><br/>
+              The data being reported does <b>not</b> contain any sensitive data like pool names, object names, object contents,
+              hostnames, or device serial numbers. It contains counters and statistics on how the cluster has been
+              deployed, the version of Ceph, the distribution of the hosts and other parameters which help the project
+              to gain a better understanding of the  way Ceph is used. The data is sent secured to https://telemetry.ceph.com.</p>
+            <div *ngIf="moduleEnabled">
+              The plugin is already <b>enabled</b>. Click <b>Deactivate</b> to disable it.&nbsp;
+              <button type="button"
+                      class="btn btn-light"
+                      (click)="disableModule('The Telemetry module has been disabled successfully.')"
+                      i18n>Deactivate</button>
+            </div>
+            <legend i18n>Channels</legend>
+            <p i18n>The telemetry report is broken down into several "channels" each with a different type of information that can
+              be configured below.</p>
+
+            <!-- Channel basic -->
+            <div class="form-group row">
+              <label class="cd-col-form-label"
+                     for="channel_basic">
+                <ng-container i18n>Basic</ng-container>
+                <cd-helper>
+                  <ng-container i18n>Includes basic information about the cluster:</ng-container>
+                  <ul>
+                    <li i18n>Capacity of the cluster</li>
+                    <li i18n>Number of monitors, managers, OSDs, MDSs, radosgws, or other daemons</li>
```

Comment:
For naming consistency in the UI:
```suggestion
                    <li i18n>Number of monitors, managers, OSDs, MDSs, object gateways, or other daemons</li>
```